pyVmomi/SoapAdapter.py

```diff
@@ -1384,6 +1384,11 @@ def _CloseIdleConnections(self):
 
          for conn, _ in idleConnections:
             self._CloseConnection(conn)
+      else:
```

Comment:
Sounds good to me. I will put he verification code in SmartConnect since it is the entry point. Is that okay?